Description: Flush stdout when generating crc headers
  When using dietlibc, the generated crc tables are truncated unless
  stdout is flushed.
Author: John David Anglin <dave.anglin@bell.net>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: not-needed
Author: John David Anglin <dave.anglin@bell.net>
Last-Update: <2014-03-30>

--- e2fsprogs-1.42.9.orig/e2fsck/gen_crc32table.c
+++ e2fsprogs-1.42.9/e2fsck/gen_crc32table.c
@@ -97,5 +97,6 @@ int main(int argc ATTR((unused)), char**
 		printf("};\n");
 	}
 
+	fflush(stdout);
 	return 0;
 }
--- e2fsprogs-1.42.9.orig/lib/ext2fs/gen_crc32ctable.c
+++ e2fsprogs-1.42.9/lib/ext2fs/gen_crc32ctable.c
@@ -119,5 +119,6 @@ int main(int argc, char **argv)
 		output_table(crc32ctable_be, BE_TABLE_SIZE, 'b');
 	}
 
+	fflush(stdout);
 	return 0;
 }
